ansible/group_vars/database-hosts.yml

```diff
@@ -36,15 +36,5 @@ idr_omero_readonly_database:
 
 postgresql_install_extensions: True
 
-# TODO: add a variable to easily enable/disable
-# TESTING ONLY!
-#postgresql_server_conf:
-#pg_stat_staments?
-#  shared_preload_libraries: "'pg_stat_statements'"
-#  pg_stat_statements.max: 20000
-#Database tuning?
-#  default_statistics_target = 1000
-#  maintenance_work_mem = 256MB
-#  checkpoint_completion_target = 0.9
-#  effective_cache_size = "{{ (ansible_memtotal_mb / 2) | int }}MB"
-#  work_mem = 64MB
+postgresql_server_conf:
+  shared_buffers: 1GB
```

Comment:
Any chance of making this a percentage of total mem instead to make it easier to test on smaller systems?

Reply:
e.g. https://github.com/ANXS/postgresql/pull/235/files#diff-0ab2a3945984f21ca36ce37d625d716cR104

Reply:
You mean using `ansible_memtotal_mb` ?

Reply:
I wonder if there's rather more risk of allocating uselessly much? Are such tiny systems a concern?

Reply:
I would think with a min & max clause.

Reply:
Fine with me. I followed @mtbc's initial investigation suggesting 1GB is sufficient for our needs (https://trello.com/c/4V4lqQre/493-mapr-is-slow). https://www.postgresql.org/docs/9.6/static/runtime-config-resource.html seems to point towards 25% as a reasonable default. I can switch to that unless someone proposes an another percentage